### PR TITLE
Fix nuspec derpage

### PR DESCRIPTION
**Pull request by Cheesebaron**
As described in #1725 there were some issues with our nuspecs. Some of it probably my fault 😄 

This should fix this 🎉

### CODE REVIEW

File: nuspec/MvvmCross.Console.Platform.nuspec [hunk 1]
```diff
@@ -15,17 +15,7 @@ This package contains the Console 'MvvmCross Library' assemblies for MvvmCross
...
 	</metadata>
 	<files>		
-		<!-- PCL -->
-		<file src="..\bin\Release\Mvx\Portable\MvvmCross.Core.*" target="lib\portable-net45+win+wpa81+wp80" />
-
-		<!-- dotnet -->
-		<file src="..\bin\Release\Mvx\Portable\MvvmCross.Core.*" target="lib\dotnet" />
-		
 		<!-- console -->
 		<file src="..\bin\Release\Mvx\Console\MvvmCross.Console.*" target="lib\net45" />
```
Comment (kjeremy) [inline nuspec/MvvmCross.Console.Platform.nuspec]: Just as a note: this looks weird to me.

Comment (Cheesebaron) [inline nuspec/MvvmCross.Console.Platform.nuspec]: Why? Core is already included in the core project which is a dependency.

Comment (kjeremy) [inline nuspec/MvvmCross.Console.Platform.nuspec]: Sorry that comment applied to the inclusion of `Console.*`.  I just didn't expect it.

**Comment by Cheesebaron** [comment]
Don't merge this yet. I see an issue. pdbsrcv are not added to nuget packages either. Need to fix that.

**Pull request merged**